### PR TITLE
refactor: point to another folder

### PR DESCRIPTION
**Pull request by Peeeekay**
Changelog picked up from commits here:

refactor: point to another folder

### CODE REVIEW

**Comment by Peeeekay** [comment]
This is for PR: https://app.zenhub.com/workspaces/engineering-636cff9fc978ceb2aac05a1d/issues/gh/kurtosis-tech/examples/45

i am planning to just remove - `simple-subpackages-example` (It doesn't providea any value) as I can use simple-api-server that @gbouv added to examples.

File: internal_testsuites/golang/testsuite/startosis_subpackage_test/startosis_remote_subpackage_test.go [hunk 1]
```diff
@@ -33,7 +33,5 @@ func TestStarlarkRemotePackage(t *testing.T) {
 	require.Nil(t, runResult.InterpretationError, "Unexpected interpretation error")
 	require.Empty(t, runResult.ValidationErrors, "Unexpected validation error")
 	require.Empty(t, runResult.ExecutionError, "Unexpected execution error")
-	require.Equal(t, "Hello world!\n", string(runResult.RunOutput))
-
-	logrus.Infof("Successfully ran Starlark Package")
+	require.Equal(t, "", string(runResult.RunOutput))
```
Comment (leoporoli) [inline internal_testsuites/golang/testsuite/startosis_subpackage_test/startosis_remote_subpackage_test.go]: you could use a constant or a ver here like `expectedOutput = ""`
and a var for the actual value `actualOutput = string(runResult.RunOutput)`

**Review by leoporoli — APPROVED** [review]

**Comment by Peeeekay** [comment]
I decided to use `quickstart` instead as it;s more static and won't change much. @leoporoli

File: internal_testsuites/golang/testsuite/startosis_subpackage_test/startosis_remote_subpackage_test.go [hunk 2]
```diff
@@ -33,7 +35,11 @@ func TestStarlarkRemotePackage(t *testing.T) {
...
 	require.Empty(t, runResult.ValidationErrors, "Unexpected validation error")
 	require.Empty(t, runResult.ExecutionError, "Unexpected execution error")
-	require.Equal(t, "Hello world!\n", string(runResult.RunOutput))
 
-	logrus.Infof("Successfully ran Starlark Package")
+	runOutputTrimmedString := strings.Trim(string(runResult.RunOutput), "\n")
+	runOutputList := strings.Split(runOutputTrimmedString, "\n")
+
+	require.Equal(t, expectedOutputLength, len(runOutputList))
+	require.Contains(t, runOutputTrimmedString, "nginx-artifact")
```
Comment (leoporoli) [inline internal_testsuites/golang/testsuite/startosis_subpackage_test/startosis_remote_subpackage_test.go]: you could use a constant for "nginx-artifact"

File: internal_testsuites/golang/testsuite/startosis_subpackage_test/startosis_remote_subpackage_test.go [hunk 3]
```diff
@@ -33,7 +35,11 @@ func TestStarlarkRemotePackage(t *testing.T) {
...
+	require.Contains(t, runOutputTrimmedString, "my-nginx-0")
```
Comment (leoporoli) [inline internal_testsuites/golang/testsuite/startosis_subpackage_test/startosis_remote_subpackage_test.go]: you could use a constant for "my-nginx-0"

**Review by leoporoli — APPROVED** [review]

**Pull request merged**